data/blog/electron-9-0.md

```diff
@@ -0,0 +1,92 @@
+   * Added `session.serviceWorkerContext` API to access basic service worker info and receive console logs from service workers. [#22313](https://github.com/electron/electron/pull/22313)
+* `app` API changes:
+   * Added a new force parameter to `app.focus()` on macOS to allow apps to forcefully take focus. [#23447](https://github.com/electron/electron/pull/23447)
+* `BrowserWindow` API changes:
+   * Added support for property access to some getter/setter pairs on `BrowserWindow`. [#23208](https://github.com/electron/electron/pull/23208)
+
+### Deprecated APIs
+
+The following APIs are now deprecated or removed:
+
+* `shell.openItem` API is now depreciated, and replaced with an asynchronous `shell.openPath API`.
+* `<webview>.getWebContents`, which was deprecated in Electron 8.0, is now removed.
+* `webFrame.setLayoutZoomLevelLimits`, which was deprecated in Electron 8.0, is now removed.
+
+## End of Support for 6.x.y
+
+Electron 6.x.y has reached end-of-support as per the project's
+[support policy](https://electronjs.org/docs/tutorial/support#supported-versions).
+Developers and applications are encouraged to upgrade to a newer version of Electron.
+
+## What's Next
+
+In the short term, you can expect the team to continue to focus on keeping up with the development of the major components that make up Electron, including Chromium, Node, and V8. Although we are careful not to make promises about release dates, our plan is release new major versions of Electron with new versions of those components approximately quarterly. The [tentative 10.0.0 schedule](https://electronjs.org/docs/tutorial/electron-timelines) maps out key dates in the Electron 10.0 development life cycle. Also, [see our versioning document](https://electronjs.org/docs/tutorial/electron-versioning) for more detailed information about versioning in Electron.
+
+For information on planned breaking changes in upcoming versions of Electron, [see our Planned Breaking Changes doc](https://github.com/electron/electron/blob/master/docs/breaking-changes.md).
+
+### Change the default of `contextIsolation` from `false` to `true` (Starting in Electron 10)
+
+Without contextIsolation, any code running in a renderer process can quite easily reach into Electron internals or an app's preload script. That code can then perform privileged actions that Electron wants to keep restricted.
+
+Changing this default improves the default security of Electron apps, so that apps will need to deliberately opt in to the insecure behaviour. Electron will depreciate the current default of `contextIsolation` in Electron 10.0, and change to the new default (`true`) in Electron 12.0.
+
+For more information on `contextIsolation`, how to enable it easily and it's security benefits please see our dedicated [Context Isolation Document](https://github.com/electron/electron/blob/master/docs/tutorial/context-isolation.md).
+
+### Remove the `nodeIntegration` flag (Starting in Electron 10)
```

Comment:
This part is still a but hand wavy and I don't want to lock in version numbers in this blog post.  Can we not mention this removal here.

Reply:
@MarshallOfSound You mean remove this section completely? Or not mention past Electron 10?

Reply:
removed the section